interface/main/backup.php

```diff
@@ -32,6 +32,10 @@
 require_once("$srcdir/acl.inc");
 require_once("$srcdir/log.inc");
 
+// Prevent scripts calling Globals from being blocked.
+//
+session_write_close();
```

Comment:
neat; reminds me of needing to do this way back in the CDR engine reports to allow the process to run without holding up rest of OpenEMR; is that why doing it here too?

Reply:
Yep, May have been issue with the Fedora gent having issues. All the background scripts that called session open where blocked while mysqldump did it's thing.